### DISCUSSION TITLE
Simple questions about the parameters setting for MMF and SMF simulation

### DISCUSSION THREAD

**Discussion by Ford666** [ankrh/BPM-Matlab#2] [Q&A] [answered]
Dear ankrh:

Hi, thanks so much for the excellent toolbox BPM-matlab. I've just got a small question. Example1 presents the simulation of MMF with off-center Gaussian beam illumination, in which the core diameter is set to be 10 um and the area length (fiber diameter) 20um, and the refractive indices of core and cladding are 1.46 and 1.45, respectively. However, we know the core diameter of MMF is relatively large (say, >50um) that allows it to guide multiple modes. Also, the difference between the refractive indices of the core and cladding is large (say 3.57 for the core made of pure silicon and 1.45 for the cladding made of silica) . I don't know whether such parameter inconsistency would has a impact? In addition, if I want to simulate the case for step-index SMF, may I ask can I accomplish it by simply changing the core/fiber diameter and refractive indies of  core/cladding in example 1?  I appreciate if you could kindly reply to me.

**Comment by ankrh** [comment, marked as answer]
Dear Shengfu

Thank you for your interest in BPM-Matlab!

BPM-Matlab propagates the field by solving the paraxial Helmholtz equation, which is a good approximation only if the angles of incidence are not too large and the differences in refractive index are also not too large. How large is “too large” is hard to say and will depend on the step size you choose in the simulations.

One thing you don’t need to worry about is how many modes are supported, since BPM-Matlab does not use modes for the beam propagation itself.

As an initial simulation you can indeed try to set the core diameter to your value (e.g. 50 µm) and the refractive indices also to your values (3.57 and 1.45) and see how the beam propagates. Make sure you try to run a few simulations with various values of dz_target, because the large difference in refractive index may require you to use quite small values.

BR Anders



From: Shengfu Cheng ***@***.***>
Sent: 5. juni 2022 07:07
To: ankrh/BPM-Matlab ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ankrh/BPM-Matlab] Simple questions about the parameters setting for MMF and SMF simulation (Discussion #2)


Dear ankrh:

Hi, thanks so much for the excellent toolbox BPM-matlab. I've just got a small question. Example1 presents the simulation of MMF with off-center Gaussian beam illumination, in which the core diameter is set to be 10 um and the area length (fiber diameter) 20um, and the refractive indices of core and cladding are 1.46 and 1.45, respectively. However, we know the core diameter of MMF is relatively large (say, >50um) that allows it to guide multiple modes. Also, the difference between the refractive indices of the core and cladding is large (say 3.57 for the core made of pure silicon and 1.45 for the cladding made of silica) . I don't know whether such parameter inconsistency would has a impact? In addition, if I want to simulate the case for step-index SMF, may I ask can I accomplish it by simply changing the core/fiber diameter and refractive indies of core/cladding in example 1? I appreciate if you could kindly reply to me.

**Reply by Ford666** [reply]
Dear ankrh:

Thanks for your detailed reply and instruction, and now I can already get my desired simulation results. Great toolkit!

Best Regards,
Shengfu
